sdk/cosmosdb/cosmos/src/common/constants.ts

```diff
@@ -181,7 +181,13 @@ export const Constants = {
 
   // Platform
   CurrentVersion: "2020-07-15",
+  /**
+ * @hidden
+ */
   AzureNamespace: "Azure.Cosmos",
+  /**
+ * @hidden
+ */
   AzurePackageName: "@azure/cosmos",
   SDKName: "azure-cosmos-js",
   SDKVersion: "3.16.3",
```

Comment:
Isn't adding @Hidden decorator also a breaking change? If it is not considered brekaing for some reason -  should we hide the SDKVersion as well? To avoid having to go through API review each time?

Reply:
done